src/renderer-process/proxy-view/ui/proxy-dashboard-header.tsx

```diff
@@ -16,18 +15,23 @@ export const ProxyDashboardHeader = ({
   ipAddress,
   isClearAllDisabled,
   isDownloadInProgress,
+  isFetchingProfiles,
+  isFetchingSuites,
   isImportHarDisabled,
   isImportHarInProgress,
   isRecording,
   onImportHarClick,
+  onSearchProfiles,
+  onSearchSuites,
   port,
+  profiles,
+  selectedProfile,
   selectedSuite,
   setFilterRegex,
   setIsDownloadInProgress,
+  onSelectProfile,
   setSelectedSuite,
   suites,
-  isFetchingSuites,
-  onSearchSuites,
 }: ProxyDashboardHeaderProps): JSX.Element => (
   <div
     className='proxy-dashboard-header'
```

Comment:
I would suggest changing `proxy-dashboard-header` from-
```
align-items: center;
display: flex;
justify-content: space-between;
  ```
  to-
  ```
  align-items: center;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  ```
  And add to `ConnectionInfoBanner` `Paper` style-
  ```
  justify-self: center;
  ```
  
  I think it looks a bit better when working w/o a monitor.